website/docs/about/contribution.md

```diff
@@ -1,34 +1,110 @@
+
+Alternatively, if you just have a question or you’re not sure your project is right for collaboration, reach out to the Design System team in [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS).
+
+### Evaluation of the request
+
+The Design System team will then evaluate each incoming request and provide the Requestor with a decision. 
+
+#### How we evaluate requests
+
+We evaluate requests based on the following criteria:
+
+- Reusability across multiple products
+- Relevance and added value for the design system as a whole
+- Integration with existing foundations, components, and features of the system
+- Impact on existing components, especially in the case of breaking changes
+- Expected effort and possible complexities and unknowns involved
+- Availability of resources on the Design System team
+- Prioritization in relation to other initiatives already undertaken and/or planned
+
+#### Possible outcomes
+
+Based on the evaluation criteria, there are different possible outcomes:
+
+- We start or plan to **dedicate time and resources to the project**. We plan a kickoff with the Requestor to further discuss the project.
+- We believe there’s an **opportunity for future collaboration**, but don’t have time or resources to dedicate to it now. We’ll continue to monitor interest in the request and re-evaluate it in the (near) future.
+- We **can’t support this request**. We’ll provide guidance and suggestions on how to move forward, but the final decision on how to proceed with the work will be left to the Requestor.
+
+### Kickoff meeting
+
+We’ll organize a Kickoff meeting with the Requestor (and any necessary stakeholders) for projects to which we can dedicate time and resources. In this Kickoff meeting, we aim to define the following:
+
+- What resources from each team will be allocated to the project, and what each individual's role is
+- What the scope of the project is (or if more discovery is necessary), and if the project can be delivered incrementally
+- What the timeline for the project is, and if there’s a deadline that needs to be met
+- What the open questions, risks, or unknowns are for this project
+- Who the stakeholders are, what their involvement is, and whose approval is required
+- What the first actionable steps and milestones are
+- How the team plans to communicate about the project
+
+### Collaborative execution
+
+The dedicated resources from each team will work together to complete the project according to the scope and timeline defined in the kickoff. This process is variable, depending on the project’s complexity, and it can be quite intensive for larger collaborations given the high bar for quality required. This project will need to go through thorough reviews with key stakeholders, as defined in the kickoff.
```

Comment:
```suggestion
The dedicated resources from each team will work together to complete the project according to the scope and timeline defined in the kickoff. This process is variable, depending on the project’s complexity, and it can be quite intensive for larger collaborations given the high bar for quality required. The project will need to go through thorough reviews with key stakeholders, as defined in the kickoff.
```